includes/repositories/EUtilsRepository.php

```diff
@@ -175,43 +177,46 @@ public function createProperty($cvterm_id, $value) {
    */
   public function createAccession($accession) {
     if (!isset($accession['db'])) {
-      throw new Exception('DB not provided for accession ' . $accession['value']);
+      if (!isset($accession['db_label'])) {
+        throw new Exception(
+          'DB not provided for accession ' . $accession['value']
+        );
+      }
+      else {
+        return NULL;
+      }
     }
 
+    // Try getting the db record with the prefix NCBI
     $db = $this->getDB('NCBI ' . $accession['db']);
```

Comment:
should this be `strtolower`?

```suggestion
    $db = $this->getDB('NCBI ' . $accession['db']);
```

Reply:
yeah would be better to remove special characters too (space, colons, commands, etc)